circle.yml

```diff
@@ -0,0 +1,31 @@
+machine:
+  environment:
+    # We need to set this variable to let Anaconda take precedence
+    PATH: "/home/ubuntu/miniconda/envs/circleenv/bin:/home/ubuntu/miniconda/bin:$PATH"
+
+dependencies:
+  cache_directories:
+    - "~/miniconda"
+  pre:
+    # Disable pyenv (no cleaner way provided by CircleCI as it prepends pyenv version to PATH)
+    - rm -rf ~/.pyenv
+    - rm -rf ~/virtualenvs
+    - wget -q http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O ~/miniconda.sh
+    - chmod +x ~/miniconda.sh
+    - ~/miniconda.sh -b -p /home/ubuntu/miniconda
+    - conda update --yes --quiet conda
+    - conda create -n circleenv --yes pip python=2.7
+    - sed -i "s/ENABLE_USER_SITE = .*/ENABLE_USER_SITE = False/g" /home/ubuntu/miniconda/envs/circleenv/lib/python2.7/site.py
+    - conda install -n circleenv --yes numpy scipy seaborn mayavi setuptools matplotlib pillow sphinx nose
+    - pip install -r requirements.txt
+
+  override:
+    - python setup.py develop
+
+test:
+  override:
+    - cd doc; make html
```

Comment:
use
make html theme=rtd

That way we build the read the docs theme, and can preview what goes online.